config.lua.dist

```diff
@@ -94,7 +94,7 @@ serverSaveClose = false
 serverSaveShutdown = true
 
 -- Rates
--- NOTE: rateExp is not used if you have enabled stages in data/XML/stages.xml
+experienceStages = false
```

Comment:
If no one objects, I think `useStages` would look nicer here. (The all caps enum on the cpp side is fine though)

Reply:
really better!